.github/workflows/master.yml

```
@@ -82,7 +82,7 @@ jobs:
           UNLOCK_APP_VERCEL_STAGING_VERCEL_ORG_ID: op://secrets/vercel/org-id
           UNLOCK_APP_VERCEL_STAGING_VERCEL_TOKEN: op://secrets/vercel/deployment-token
           UNLOCK_APP_VERCEL_STAGING_NEXT_PUBLIC_BASE64_WEDLOCKS_PUBLIC_KEY: op://secrets/wedlocks/public-key
-          UNLOCK_APP_VERCEL_STAGING_NEXTHAUTH_URL: 'https://unlock-protocol.com'
+          UNLOCK_APP_VERCEL_STAGING_NEXTAUTH_URL: 'https://unlock-protocol.com'
```

This would be more accurate?
```suggestion
          UNLOCK_APP_VERCEL_STAGING_NEXTAUTH_URL: 'https://staging-app.unlock-protocol.com'
```

---

Also please add to the `pull-request.yml` and `production` workflows!